packages/docs/src/pages/index.md

```diff
@@ -3,6 +3,13 @@ title: Introduction
 weight: -100
 ---
 
+<div class="ds-c-alert">
+  <div class="ds-c-alert__body">
+    <h3 class="ds-c-alert__heading">Tell us how the system is working for you</h3>
+    <p class="ds-c-alert__text">We're conducting a survey of design system users in order to understand how they use it, what’s working well, and where it has room to grow. <br><a href="https://forms.cms.gov/cms-wide-design-system-survey" target="_blank">Take the survey</a></p>
```

Comment:
I'd like to add a `rel="noopener noreferrer"` attribute to the link here. Even though we control the survey link, it's a best-practice for links that are opened via `target="_blank"`.

If you're curious: https://developers.google.com/web/tools/lighthouse/audits/noopener